ovos_date_parser/dates_gl.py

```
@@ -0,0 +1,305 @@
+        'centuries': "seculos",
+        'millenniums': "milenios"
+    }
+
+    patron = r"(?P<value>\d+(?:\.?\d+)?)(?:\s+|\-){unit}[s]?"
+
+    for (unit_en, unit_gl) in unidades_tempo.items():
+        patron_unidade = patron.format(unit=unit_gl[:-1])  # eliminar 's' da unidade
+        unidades_tempo[unit_en] = 0
+
+        def substitucion(match):
+            unidades_tempo[unit_en] += float(match.group("value"))
```

Ruff (0.8.2)

Function definition does not bind loop variable `unit_en`

(B023)